src/modules/dashboard/lists/BorrowAssetsList/BorrowAssetsListItem.tsx

```diff
@@ -28,8 +29,11 @@ export const BorrowAssetsListItem = ({
   isFreezed,
 }: DashboardReserve) => {
   const { openBorrow } = useModalContext();
-  const { currentMarket } = useProtocolDataContext();
-  const borrowButtonDisable = isFreezed || Number(availableBorrows) <= 0;
+  const { currentMarket, currentMarketData } = useProtocolDataContext();
+
+  const isPolygonV2 = currentMarketData.chainId === ChainId.polygon && !currentMarketData.v3;
```

Comment:
I think this needs to only be applied to disabled assets array instead of all polygon v2 assets